src/Psalm/Internal/MethodIdentifier.php

```diff
@@ -13,6 +13,9 @@ class MethodIdentifier
      */
     public function __construct(string $fq_class_name, string $method_name)
     {
+        // remove trailing backslash if it exists
```

Comment:
The comment is wrong... it removes a leading backslash, no?

Reply:
Could this just be a `ltrim(...)` instead?

Reply:
> The comment is wrong... it removes a leading backslash, no?

Oh yeah, I just moved that code block, the comment was already wrong :) 

> Could this just be a ltrim(...) instead to ease reading...?

That would remove multiple backslashes which would make Psalm thing that `\\Foo` is valid but it's not.